src/Chapter20/Listing20.03.AsynchronousHighLatencyUsingTaskAsynchronyPattern.cs

```diff
@@ -24,13 +25,14 @@ public static async Task Main(string[] args)
                 url = args[1];
                 // Ignore additional parameters
             }
-            Console.Write($"Searching for '{findText}' at URL '{url}'.");
+            Console.WriteLine(
+                $"Searching for '{findText}' at URL '{url}'.");
 
             using WebClient webClient = new WebClient();
             Task<byte[]> taskDownload =
                 webClient.DownloadDataTaskAsync(url);
 
-            Console.Write("\nDownloading...");
+            Console.Write("Downloading...");
```

Comment:
```suggestion
            Console.WriteLine("Downloading...");
```
Use WriteLine over Write

Reply:
In this case we can't do this because the code is expected to append periods to the current line.

Reply:
Ah, good catch, I didn't see that later on, will remove the suggestion.